tests/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -3,13 +3,13 @@ set -eux
 
 # Folder created for specific capi release when running
 # ${CLUSTER_API_REPO}/cmd/clusterctl/hack/create-local-repository.py
-export CAPIRELEASE_HARDCODED="v1.2.99"
+export CAPIRELEASE_HARDCODED="v1.3.99"
 
 export CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
-export CAPI_REL_TO_VERSION="v1.2.2"
+export CAPI_REL_TO_VERSION="v1.3.1"
```

Comment:
upgrade tests were still using old CAPI/CAPM3 release versions, so had to fix this too